sympy/printing/octave.py

```diff
@@ -392,6 +392,10 @@ def _print_lowergamma(self, expr):
                                               self._print(expr.args[0]))
 
 
+    def _print_sinc(self, expr):
+        return "sinc(%s/pi)" % self._print(expr.args[0])
```

Comment:
Space needed before and after the division operator.
Follow PEP 8 styling guide [Link](https://www.python.org/dev/peps/pep-0008/).